.changelogs/filter_parsed_engagements.yml

```
@@ -0,0 +1,7 @@
+significance: patch
+type: fixed
+links:
+  - "https://github.com/gocodebox/lifterlms-integration-twilio/issues/32"
```

```suggestion
  - "gocodebox/lifterlms-integration-twilio#32"
```

Only short references are considered valid:
+ `#123` (same repo)
+ `gocodebox/repo#123` different repo

Validate with: `npm run dev changelog validate`